package-lock.json

```diff
@@ -1,46 +1,68 @@
 {
   "name": "RapidAPI-testing-github-action",
-  "version": "1.0.0",
-  "lockfileVersion": 1,
+  "version": "0.0.3",
```

Comment:
wait why was it 1.0.0?

Reply:
My suggestion, just to get start off on a normal version number. This PR does not warrant a major version change, but we probably should have started with 1.x so I figured we would just move it now